config/spring19_seminar/cola_var/bert.conf

```diff
@@ -0,0 +1,40 @@
+// Import the defaults using the relative path
+include "../../final.conf"
```

Comment:
Heads up for @HaokunLiu @PterosDiacos: Coordinate on whether to include final or defaults. The differences are pretty light, so either one should be okay.

Reply:
This config seems pretty good. Since the experiments I run are all on BERT, can I simply import from this one?

Reply:
Sounds good to me!